scripts/minishift/install.sh

```diff
@@ -1,28 +1,47 @@
 #!/bin/bash
 
+case "$1" in
+    ""|"--remote")
+        BASE="https://raw.githubusercontent.com/fabric8-launcher/launcher-openshift-templates/master"
+        ;;
+    "--local")
+        BASE=
+        ;;
+    "--delete")
+        oc delete project launcher
+        exit
+        ;;
+    *)
+        echo "Invalid argument: $1"
+        echo "Usage: install [--local] [--remote] [--delete]"
+        exit
+        ;;
+esac
+
 echo This script will install the Launcher in Minishift. Make sure that:
 echo 
 echo  - Minishift is running 
 echo  - You have run oc login previously
 echo  - Your GitHub Username is correct [found from git config github.user]: $(git config github.user)
 echo  - Your GitHub Token is correct [found from git config github.token]: *REDACTED*
 echo 
-echo Press any key to continue ...
+echo Press ENTER to continue ...
 read 
 
 echo Creating launcher project ...
 oc new-project launcher
 
 echo Processing the template and installing ...
-oc process --local -f openshift/launcher-template.yaml \
+curl -s $BASE/released.properties | oc process --local -f $BASE/openshift/launcher-template.yaml \
    LAUNCHER_KEYCLOAK_URL= \
    LAUNCHER_KEYCLOAK_REALM= \
    LAUNCHER_MISSIONCONTROL_GITHUB_USERNAME=$(git config github.user) \
    LAUNCHER_MISSIONCONTROL_GITHUB_TOKEN=$(git config github.token) \
    LAUNCHER_MISSIONCONTROL_OPENSHIFT_CONSOLE_URL=$(minishift console --url | sed 's/\/console//') \
-   --param-file=released.properties -o yaml | oc create -f -
+   --param-file - -o yaml | oc create -f -
```

Comment:
Didn't know that was possible.

Reply:
Me neither, I first tried putting a URL in there but that didn't work and then I thought that most UNIX commands support `-` for filenames to indicate STDIN. And that worked :-)